mkosi/__init__.py

```diff
@@ -2042,7 +2042,7 @@ def invoke_dnf(
     cmdline += [command, *sort_packages(packages)]
 
     with mount_api_vfs(args, root):
-        run(cmdline, env=dict(KERNEL_INSTALL_BYPASS="1"))
+        run(cmdline, env=dict(SOURCE_DATE_EPOCH="0", KERNEL_INSTALL_BYPASS="1"))
```

Comment:
I don't understand. Why would we want to pass "0" ?

Reply:
It's a draft, this can be changed to CURRENT_TIME or the manifest timestamp. `0` is just a simple default to get things working.